lib/managers/KalturaFetchManager.js

```diff
@@ -262,6 +263,8 @@ class KalturaFetchManager extends kaltura.KalturaManager
 						{
 							engine.transcodeFile(commandLine).then(
 								function (data){
+									if(KalturaConfig.config.h264Verfication.enabled == 1)
```

Comment:
I see that in all cases you use this after transcoding a file - why not inserting this into the Transcoding Engine ? Or Create a transcoding wrapper